Client/TabManagement/Tab.swift

```diff
@@ -976,7 +976,10 @@ class TabWebView: WKWebView, MenuHelperInterface {
             CreditCardHelper.focusNextInputField(tabWebView: self,
                                                  logger: self.logger)
         }
-        accessoryView.doneClosure = { self.endEditing(true) }
+
+        accessoryView.doneClosure = {
+            CreditCardHelper.blurActiveElement(tabWebView: self, logger: self.logger)
+            self.endEditing(true) }
```

Comment:
nit, parenthesis position
```suggestion
            self.endEditing(true) 
        }
```

Reply:
oof, missed this